requirements/deploy.txt

```diff
@@ -1,8 +1,8 @@
 -r pip.txt
 # http://initd.org/psycopg/docs/install.html#binary-install-from-pypi
 psycopg2==2.8.3 --no-binary psycopg2
-gunicorn==19.9.0
+gunicorn==20.0.4
```

Comment:
Nothing breaking


    fix binding a socket using the file descriptor
    remove support for the bdist_rpm build